public/docs/js/latest/guide/cheatsheet.jade

```diff
@@ -0,0 +1,9 @@
+.banner
+  p.text-body This cheat sheet is provisional and may change. Angular 2 is currently in Developer Preview.
+
+article(class="l-content-small grid-fluid docs-content")
+  .cheatsheet
+    h2 Angular for JavaScript Cheat Sheet (v2.0.0-alpha.44)
```

Comment:
We probably need a more automated way of specifying the version number of Angular in these (all three) cheatsheet.jade files